index.js

```diff
@@ -57,12 +57,16 @@ function resolveUrlLoader(content, sourceMap) {
   loader.cacheable();
 
   // incoming source-map
-  var sourceMapConsumer, contentWithMap;
+  var sourceMapConsumer, contentWithMap, sourceRoot;
   if (sourceMap) {
 
     // sass-loader outputs source-map sources relative to output directory so start our search there
     try {
       relativeToAbsolute(sourceMap.sources, outputPath, resolvedRoot);
+      // There are now absolute paths in the source map so we don't need it anymore
+      // However, later when we go back to relative paths, we need to add it again
+      sourceRoot = sourceMap.sourceRoot;
+      sourceMap.sourceRoot = null;
     } catch (exception) {
```

Comment:
Should we add some tests, so that we can make sure, that we cover most of the different use cases to not mess up someones repo?